### PR TITLE
Used select statement for async go

### PR DESCRIPTION
**Pull request by itaditya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +243,40 @@ func fetchConcurrent() {
...
+		c <- fetchA()
+	}(aChan)
+	go func(c chan fetchResult) {
+		c <- fetchB()
+	}(bChan)
+	go func(c chan fetchResult) {
+		c <- fetchC()
+	}(cChan)
+	
+	for i := 0; i < 2; i++ {
```
Comment (pazams) [inline README.md]: should be `i < 3`

File: README.md [hunk 2]
```diff
@@ -243,6 +243,40 @@ func fetchConcurrent() {
...
+        select {
+			case a := <-aChan:
+				fmt.Println(a)
+			case b := <-bChan:
+				fmt.Println(b)
+			case c := <-cChan:
+				fmt.Println(c)
+			
+        }
+	}
```
Comment (pazams) [inline README.md]: Please format with `gofmt`. 
When online, it's also easy to use https://play.golang.org/ to format snippets.

Comment (itaditya) [reply README.md]: Ok thanks for the tip

File: README.md (hunk 1 above)
Comment (itaditya) [reply README.md]: oops

File: README.md (hunk 2 above)
Comment (itaditya) [reply README.md]: I'll update soon

**Pull request merged**